### PR TITLE
fix: make withdrawFee undefined by default

### PR DESCRIPTION
**Pull request by bodymindarts**

### CODE REVIEW

**Review by vindard — DISMISSED** [review]
Approved, assuming there isn't some acount-level setting we still wanted enforced outside of what comes from global `feeConfig`

**Review by dolcalmi — CHANGES_REQUESTED** [review]

**Review by dolcalmi — COMMENTED** [review]
will we have a migration after this? default is for creation

File: core/api/test/unit/payments/onchain-payment-flow-builder.spec.ts [hunk 1]
```diff
@@ -821,7 +821,7 @@ describe("OnChainPaymentFlowBuilder", () => {
                           )
 
                     const minBankFee = paymentAmountFromNumber({
-                      amount: senderAccount.withdrawFee,
+                      amount: senderAccount.withdrawFee as Satoshis,
```
[the remark this answers is not in the document]
Comment (bodymindarts) [reply core/api/test/unit/payments/onchain-payment-flow-builder.spec.ts:824]: This is not coming from config - its set on the dummy `senderAccount` definer in line 69.

File: core/api/src/services/mongoose/schema.ts [hunk 2]
```diff
@@ -151,11 +149,11 @@ const AccountSchema = new Schema<AccountRecord>(
       required: true,
       default: () => crypto.randomUUID(),
     },
-
     withdrawFee: {
       type: Number,
-      default: feesConfig.withdrawDefaultMin,
+      default: undefined,
```
[the remark this answers is not in the document]
Comment (bodymindarts) [reply core/api/src/services/mongoose/schema.ts]: removed

**Comment by bodymindarts** [comment]
> will we have a migration after this? default is for creation

Yes there will be a migration to remove all persisted withdraw fees.

**Pull request merged**